packages/graphql/src/resolvers/marketplace/listings.js

```diff
@@ -98,6 +102,15 @@ async function resultsFromIds({ after, ids, first, totalCount, fields }) {
   }
 }
 
+async function getEvents(args) {
```

Comment:
I think we currently have 5K events on marketplace contract... that's going to be lots of events.
Not sure if it makes sense to expose this without pagination. Perhaps add a comment for later ?

Reply:
This is just a helper method that expects filters to be passed in `args`. Adding a comment...